cost/google/idle_vm_recommendations/google_vm_recommendations.pt

```diff
@@ -560,6 +562,12 @@ policy "policy_recommendations" do
       field "service" do
         label "Service"
       end
+      field "accountID" do
+        label "Account ID"
+      end
```

Comment:
Should we add a standard `accountName` field?

Reply:
done